default/python/AI/ProductionAI.py

```diff
@@ -710,7 +710,7 @@ def generate_production_orders():
                     use_sys = ([(-1, INVALID_ID)] + sorted([(dist, sys_id) for sys_id, dist in distance_map.items()]))[:2][-1][-1]  # kinda messy, but ensures a value
                 if use_sys != INVALID_ID:
                     try:
-                        use_loc = state.get_empire_planets_by_system(include_outposts=True)[use_sys][0]
+                        use_loc = state.get_empire_planets_by_system(use_sys, include_outposts=True)[0]
```

Comment:
`use_sys, include_outposts=True` Please call arguments in same way as them declared - > `sys_id=use_sys, include_outposts=True`  here and in other places.

Reply:
I understand where you are coming from but I don't agree with you here.

The ```sys_id=``` adds no information to the call as the meaning of the parameter is pretty clear from context. Additionally, it makes the code less readable because it isn't unusual to have some local ```sys_id``` variable that is not supposed to be passed here:
```
sys_id = 1
use_sys = 2
state.get_empire_planets_by_system(sys_id=use_sys)
state.get_empire_planets_by_system(use_sys)
```
Second call is much clearer for me and makes navigating the code easier as well.

I am open to change it if you have a good argument for your case.

Reply:
One of them looks like error:
```
state.get_empire_planets_by_system(use_sys)
state.get_empire_planets_by_system()
```